src/java.desktop/windows/classes/sun/awt/windows/WTrayIconPeer.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2005, 2018, Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2005, 2022, Oracle and/or its affiliates. All rights reserved.
```

Comment:
I'm for keeping the expanded imports in WTrayIconPeer.java, even it's the only change left now.

I'd probably remove the unused import from `TrayIcon.java` too. However, the change in this case is too minor.

Reply:
My preference would beto revert it. Once all of the actual changes associated with this fix were reverted, fixing the imports (or any similar type cleanup) in a file that otherwise doesn't need to be touched at all becomes an unrelated change. Maybe Phil can weigh in on this point.

Reply:
Your approach makes the changeset _cleaner_, I agree. Nothing's changed in these two files. Better to avoid touching them and creating confusion. Let's leave these clean-up changes until the files are modified.